### PR TITLE
Add Copyright Headers to generated files

### PR DESCRIPTION
**Pull request by awharn**
Adds copyright headers to all files trimmed out of the command tree that remain in our repository.
Also adds contribution instructions and documents these changes.
Also adds PDF generation of the web help.

Signed-off-by: Andrew W. Harn <andrew.harn@broadcom.com>

### CODE REVIEW

**Review by t1m0thyj — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]
PR looks great as it is.
Shouldn't have waited for a second review 😋 

I do have a question, do we want to update the zowe.template.json file to jsonc and add the license_header to it...

Also, should we update the `updateLicense` script to include any jsonc files as well ?

**Comment by zFernand0** [comment]
After discussing IRL, I don't think we need to do anything with the template file or the existing updateLicense script 😋 
Thank you both for your patience 😋

**Pull request merged**
